### PR TITLE
Randomize the default order we use to call bidders by default.

### PR DESCRIPTION
**Pull request by dbemiller**
## Type of change
Feature

## Description of change
Randomizing the default order we use to call bidders, per #1335

This is an API breaking change... but IMO it's not really worth waiting 'til 1.0 for. It matches the project's values much better, and support for it seems pretty universal.

### CODE REVIEW

File: src/constants.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
     "bidWon": "adUnitCode"
   },
   "ORDER": {
-    "RANDOM": "random"
+    "fixed": "fixed",
```
Comment (mkendall07) [inline src/constants.json]: now that we have language `const` support the value of a `constants` file feels diminished. Probably better to define these constant strings closer to where they are used in the code.

Comment (dbemiller) [inline src/constants.json]: that..... makes so much more sense. Will fix shortly.

**Comment by mkendall07** [comment]
need approval from @mjacobsonny and @bretg  before merge.

**Comment by mjacobsonny** [comment]
LGTM.  I'm fine with this as long as users have the ability to configure fixed bidder order to replicate current state.

**Comment by dbemiller** [comment]
Yeah. You get the current behavior if you call `$$PREBID_GLOBAL$$.setBidderSequence('fixed')`

**Comment by bretg** [comment]
Approved, though please highlight this change in the release notes so Pubs know to add setBidderSequence('fixed') as desired.

**Comment by dbemiller** [comment]
Will do. Thanks ^^

**Pull request merged**